discovery-provider/src/queries/get_health.py

```diff
@@ -304,7 +304,9 @@ def get_health(args: GetHealthArgs, use_redis_cache: bool = True) -> Tuple[Dict,
         db_connections_json, db_connections_error = _get_db_conn_state()
         health_results["db_connections"] = db_connections_json
         health_results["latitude"] = shared_config["serviceLocation"]["serviceLatitude"]
-        health_results["longitude"] = shared_config["serviceLocation"]["serviceLongitude"]
+        health_results["longitude"] = shared_config["serviceLocation"][
```

Comment:
not sure why this changed. was it some auto formatter?

Reply:
I think the line may have been too long. I'm using the autoformatter settings here: https://github.com/AudiusProject/audius-protocol/blob/17aa642d6c74f46afddb75a7734387142bf18dd3/discovery-provider/.vscode/settings.json.

Reply:
okay awesome!